### ISSUE TITLE
Add data for new Canadian regions

### ISSUE THREAD

**Issue by mpinsky** [pinskylab/OceanAdapt#132]
To do
- Try to get data from each region. Also Alaska, SEUS, and Scotian Shelf 2019.
- Develop code to clean the data, in partnership with data providers (do this on a separate branch)
- Add to OceanAdapt
[How to Access Research Trawl Survey Data.docx](https://github.com/pinskylab/OceanAdapt/files/4765215/How.to.Access.Research.Trawl.Survey.Data.docx)

**Comment by df511** [comment]
Gulf Region; link to some trawl surveys located here:https://open.canada.ca/data/en/dataset/1989de32-bc5d-c696-879c-54d422438e64

**Comment by df511** [comment]
Pacific Region; Shellfish Surveys - Leslie Barton (Leslie.Barton@dfo-mpo.gc.ca) stated that they are hoping to publish the Small Mesh Multi-species surveys (shellfish survey data) later this year. She is currently waiting on publication of a technical report, containing these data. Check back in late fall/early winter.

**Comment by df511** [comment]
Re: Alaska, SEUS, and Scotian Shelf 2019
Alaska - Jim Thorson on parental leave, try to reach out to other NOAA contacts.
SEUS - 2019 data now available (!) at https://www2.dnr.sc.gov/seamap/Account/LogOn?ReturnUrl=%2fseamap%2fReports 
Scotian Shelf (SCOT) - 2019 available directly from Mike McMahon via ftp://ftp.dfo-mpo.gc.ca/MarPED/RVSurvey_20200420.zip; however, formatted very differently than previous source. This update is on hold; however, check back at original link periodically (https://gisp.dfo-mpo.gc.ca/arcgis/rest/services/FGP)

**Comment by df511** [comment]
Central and Arctic Region;
Contacted Margaret Treble (Margaret.Treble@dfo-mpo.gc.ca) to request Groundfish Survey data (1999 - present) and she responded after follow-up on 7/21/20 saying they cannot make those data available at this time. Will **check back in late fall, or winter**.

Contacted Wojciech Walkusz (Wojciech.Walkusz@dfo-mpo.gc.ca) in early July, and sent a follow up on 7/21/20, regarding data collected from the multi-species offshore trawl survey of Davis Strait and Ungava Bay (Shrimp Fishing Areas 0, 1, 2ex, & 3). **Waiting on their reply!**

**Comment by df511** [comment]
Quebec Region;

Data (1978 - present) available at the following four open data links:
https://open.canada.ca/data/en/dataset/d4ec2d6b-f4bc-4c6c-b866-b26e507a3b76
https://open.canada.ca/data/en/dataset/f1fc359c-0ed1-4045-a421-adef2497b68d
https://open.canada.ca/data/en/dataset/267e20aa-97e8-43da-8c23-1234376938bc
https://open.canada.ca/data/en/dataset/182acf27-3ca0-49bb-9a68-50dbd8657331

Additional data from the Sentinel Fisheries Program (1994 - present) available here:
https://ogsl.ca/fr/biodiversite/contexte

**Comment by df511** [comment]
**Maritimes Region**;

Summer and Winter surveys available via .Rdata files at ftp.dfo-mpo.gc.ca/MarPED/
-> press enter as "Registered User" (password remains blank)
-> then navigate to "RVSurvey_<YYYMMDD>.zip"

**Comment by df511** [comment]
**Newfoundland Region;**

Brittany no longer works in the position responsible for managing these data, and the position has yet to be filled, but she said she will forward our request to her supervisor. **Data request form sent to Brittany Pye (Brittany.Pye@dfo-mpo.gc.ca)on 7/22.**

**Comment by df511** [comment]
**Scotian Shelf Region**

Updated publicly available links in csv format! 

4VSW: https://open.canada.ca/data/en/dataset/a851ce30-e216-4d7d-a29c-05631eef140e

Fall: https://open.canada.ca/data/en/dataset/5f82b379-c1e5-4a02-b825-f34fc645a529

Spring: https://open.canada.ca/data/en/dataset/fecf045a-95a2-4b69-8a40-818649a62716

Summer: https://open.canada.ca/data/en/dataset/1366e1f1-e2c8-4905-89ae-e10f1be0a164

**Comment by mpinsky** [comment]
For Newfoundland: @df511  to email Nadine Wells again. If no response, @mpinsky  to email DFO and Nadine.
